proxy/types.go

```diff
@@ -277,11 +281,17 @@ func GetServiceFromProvider(provider ServiceParameterProvider) *Service {
 	if len(provider.GetString("serviceDomain")) > 0 {
 		sr.ServiceDomain = strings.Split(provider.GetString("serviceDomain"), ",")
 	}
-	if len(provider.GetString("addHeader")) > 0 {
-		sr.AddHeader = strings.Split(provider.GetString("addHeader"), ",")
+	if len(provider.GetString("addReqHeader")) > 0 {
```

Comment:
Keep the previous conditional with `addHeader` so that existing users do not get affected with the change. It can be the `else if` statement. Add a TODO note that it is deprecated and should be removed in the future. A similar logic should be applied to `setReqHeader` as well.